lib/global_config.js

```diff
@@ -157,3 +159,52 @@ const rest = {
   HTTPS_PROTOCOL: 'https'
 };
 exports.rest = rest;
+
+/**
+ * As a default we set parameters values identical like in fast-xml-parser lib defaults
+ * thus preserving backward compatibility if customer doesn't set custom configuration
+ */
+const defaultXmlParserConfiguration = {
+  ignoreAttributes: true,
+  alwaysCreateTextNode: false,
+  attributeNamePrefix: '@',
+  attributesGroupName: null
+};
+
+let xmlParserConfiguration = null;
```

Comment:
let's set it to default and in getter just return this variable?

Reply:
ok